selenium/src/main/java/tech/cscheer/impfen/selenium/Mailer.java

```diff
@@ -0,0 +1,55 @@
+/*
+   Copyright 2021 Sebastian Knabe
+ */
+package tech.cscheer.impfen.selenium;
+import java.util.Properties;
+
+import javax.mail.Authenticator;
+import javax.mail.Message;
+import javax.mail.Multipart;
+import javax.mail.PasswordAuthentication;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeBodyPart;
+import javax.mail.internet.MimeMessage;
+import javax.mail.internet.MimeMultipart;
+
+public class Mailer {
+    private static final Properties PROPERTIES = new Properties() {{
+        put("mail.smtp.auth", true);
+        put("mail.smtp.starttls.enable", "true");
+        put("mail.smtp.host", "posteo.de");
```

Comment:
Das vielleicht noch in ne Propoerty oder die config file auslagern, sonst sieht das gut aus